wallet/src/de/schildbach/wallet/ui/SetPinActivity.kt

```diff
@@ -115,6 +117,23 @@ class SetPinActivity : InteractionAwareActivity() {
         pinRetryController = PinRetryController.getInstance()
 
         walletApplication = application as WalletApplication
+
+        if (walletApplication.wallet == null) {
+            val dialog = FancyAlertDialog.newInstance(
+                R.string.set_pin_error_missing_wallet_title,
+                R.string.set_pin_error_missing_wallet_message, R.drawable.ic_error,
+                R.string.button_ok,
+                R.string.button_cancel
+            )
+            val fancyAlertDialogViewModel = ViewModelProvider(this)[FancyAlertDialogViewModel::class.java]
+            fancyAlertDialogViewModel.onPositiveButtonClick.observe(this) {
+                alertDialog = ReportIssueDialogBuilder.createReportIssueDialog(this,
+                    WalletApplication.getInstance()).buildAlertDialog()
+                alertDialog.show()
+            }
+            dialog.show(supportFragmentManager, "serious_error_dialog")
+        }
+
         if (walletApplication.wallet.isEncrypted) {
```

Comment:
This will display a dialog that tells the user to contact support.  What if wallet is `null` will the dialog be displayed? or will it close because the app crashes on line 137?